courses/free-cources-en

```diff
@@ -0,0 +1,13 @@
+### Index
+
+-   [Programming-with-JS](#Programming-with-JS)
+
+-   [ML-For-Beginners](#ML-For-Beginners)
+
+### Web-Dev-for-Beginners
+
+-   [Programming-with-JS](https://github.com/microsoft/Web-Dev-For-Beginners) - Microsoft
```

Comment:
This can be moved [here](https://github.com/EbookFoundation/free-programming-books/blob/main/books/free-programming-books-langs.md)